r_basics_visualize_data/r_basics_visualize_data.md

```diff
@@ -0,0 +1,843 @@
+
+* In RStudio, open a new project (File, New Project)
+* Select Version Control, then Git
+* Drop this link into the "Repository URL": https://github.com/arcus/education_r_environment
+* Change the "Project directory name" and "Create project as a subdirectory of" boxes to suit your needs (where will this code be stored on your computer?).
+* Click to select the "Open in new session" checkbox
+* Click "Create Project"
+* In the file area to the lower right, you'll see, among multiple choices, the folder called "r\_basics\_visualize\_data".  That's the code for this module!
+
+**Want to watch this process?  Click on the image below to play an animated gif.  It will continue to loop and you can re-start it by clicking again.**
+
+<div style="display:none">@gifPreload</div>
+
+<figure>
+  <img src="https://github.com/arcus/education_modules/blob/r_basics_visualize_data/r_basics_visualize_data/media/rstudio_new_project.png?raw=true" height="384" width="512" alt="RStudio can create a new project that gets its contents from a git repository." data-alt="https://github.com/arcus/education_modules/blob/r_basics_visualize_data/r_basics_visualize_data/media/rstudio_new_project.gif?raw=true" style = "border: 1px solid rgb(var(--color-highlight));">
+
+<figcaption style = "font-size: 1em;">Click on the image to play the demo of the above steps!</figcaption>
+</figure>
+
+If you already completed this work for a previous module, and it's been a while since you downloaded this project to your computer, you may want to get any new and improved files that have been placed there in the meantime:
+
+* Open your project.
+* In the Version Control menu, choose "pull branches".  There are two places to do this, as shown below:
+
+![Version control button in RStudio menu at top of window drops down with choices to pull and push branches.](media/pull_branches.png)<!-- style = "border: 1px solid rgb(var(--color-highlight)); max-width: 400px;"-->
+
+![Tools tab in the highest level menu bar drops down with choices to pull and push branches.](media/pull_branches_2.png)<!-- style = "border: 1px solid rgb(var(--color-highlight)); max-width: 400px;"-->
+
+<div class = "warning">
+If you're pulling branches after having worked in previous modules, you might have made local changes (for example, when you filled in exercise code) that will be overwritten by pulling the latest version.  If you want to save your changes, consider making a copy of any exercise files and naming them something new.  For example, if you have already worked in the `r_basics_transform_data` exercise files, you might want to save your version of `transform_exercises.Rmd` to `my_transform_exercises.Rmd`.  That way, you can pull down the latest version of code, overwriting `transform_exercises.Rmd` while holding on to your changes in the new file.
+</div>
+

```

Comment:
I think it's tricky to break up the wall of text since the desired outcome is one of the two things on this page, and splitting it into two pages makes it feel like two independent tasks.  I'll experiment with "accordion"-ing the content but in the interest of getting this into production I'm not going to worry about getting this fixed rn.